packages/paste-core/components/chat-log/stories/index.stories.tsx

```diff
@@ -28,31 +33,172 @@ export const OutboundChatMessage = (): React.ReactNode => {
   );
 };
 
+export const InboundMessageWithMeta = (): React.ReactNode => {
+  return (
+    <>
+      <ChatMessage variant="inbound">
+        <ChatBubble>
+          Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
+        </ChatBubble>
+        <ChatMessageMeta aria-label="said by Gibby Radki 4 minutes ago">
+          <MediaObject verticalAlign="center">
+            <MediaFigure spacing="space30">
+              <Avatar name="Gibby Radki" size="sizeIcon20" />
+            </MediaFigure>
+            <MediaBody>Gibby Radki</MediaBody>
+          </MediaObject>
+          4 minutes ago
+        </ChatMessageMeta>
+      </ChatMessage>
+    </>
+  );
+};
```

Comment:
ℹ️ No required change here, just wanted to point this out.
I like to omit the curly braces when the arrow function only has a return statement, can save some lines in the long run.

```suggestion
export const InboundMessageWithMeta = (): React.ReactNode => (
  <>
    <ChatMessage variant="inbound">
      <ChatBubble>
        Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
      </ChatBubble>
      <ChatMessageMeta aria-label="said by Gibby Radki 4 minutes ago">
        <MediaObject verticalAlign="center">
          <MediaFigure spacing="space30">
            <Avatar name="Gibby Radki" size="sizeIcon20" />
          </MediaFigure>
          <MediaBody>Gibby Radki</MediaBody>
        </MediaObject>
        4 minutes ago
      </ChatMessageMeta>
    </ChatMessage>
  </>
);
```